test/revert-cleanup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Revert Ganache to Pre-Setup"
+cd dex-contracts
```

Comment:
I don't really like that this `cd` is matched by a `cd` in another file. It will be hard to reason about which script starts in which folder it is expecting to be and in which folder it will end up. It would be nice if all scripts can expect to be called from some folder and will return to that folder at the end.

Reply:
I agree. I also never really liked that. I will take you suggestion and try to make that happen!